.github/actions/setup-zui/action.yml

```diff
@@ -8,10 +8,13 @@ runs:
       with:
         go-version: '1.21'
 
+    # Caching is disabled because it resulted in getting x86_64 Zed binaries
+    # on arm64 builds. See:
+    # - https://github.com/actions/setup-node/issues/1008
     - name: Install Node
       uses: actions/setup-node@v3
       with:
-        cache: yarn
+        # cache: yarn
```

Comment:
In an early test iteration, @mattnibs spotted that the `zed` binaries bundled with Zui were still showing up as Intel. I ultimately traced it to a side effect of this caching, then found open issue https://github.com/actions/setup-node/issues/1008 that seems to indicate this is a known problem. I briefly went down the path of trying to find another way to keep caching in play via some other config, but I also noticed that disabling caching wasn't slowing down the build process much at all. In the interest of simplicity I figured we could just live without it for now. I'm Subscribed to that issue so if/when it gets addressed I could revisit this. In the meantime I've included this comment so someone doesn't turn it on again without recognizing the effect it'll have on the macOS builds.